v03_pipeline/lib/model/environment.py

```diff
@@ -54,4 +61,5 @@ class Env:
     PROJECT_ID: str | None = PROJECT_ID
     REFERENCE_DATASETS_DIR: str = REFERENCE_DATASETS_DIR
     SHOULD_REGISTER_ALLELES: bool = SHOULD_REGISTER_ALLELES
+    SHOULD_TRIGGER_HAIL_BACKEND_RELOAD: bool = SHOULD_TRIGGER_HAIL_BACKEND_RELOAD
```

Comment:
to confirm, we don't have to set this env var because we're not using this functionality

Reply:
Yes that is correct!  At least for now!